src/lib/homestore_backend/hs_shard_manager.cpp

```diff
@@ -303,6 +303,36 @@ void HSHomeObject::on_shard_message_rollback(int64_t lsn, sisl::blob const& head
     }
 }
 
+// FIXME: Bugfix in progress from Hooper, will fix later.
```

Comment:
NIT: Already fixed, it can be removed now. 😄

Reply:
Thx, removed in latest push.